ui/src/components/dagComponent.tsx

```
@@ -21,91 +21,112 @@ const DagComponent = ({ dag }: Props) => {
+            )}
+            <p>
+              <strong class="font-medium text-gray-300">Image:</strong>{" "}
+              {taskDetails()?.image}
+            </p>
+            <p>
+              <strong class="font-medium text-gray-300">Parameters:</strong>
+            </p>
+            <ul class="ml-4 list-disc text-gray-400">
+              {taskDetails()?.parameters &&
+                taskDetails()!.parameters.map((param, index) => (
+                  <li>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)